src/commands/svc/svc-process.ts

```diff
@@ -0,0 +1,44 @@
+import { Command } from '@oclif/core'
+import { ProcessExecutionResult } from '../../services/process-executor-service'
+
+export enum ServiceTypes {
+  START = 'Starting',
+  STOP = 'Stopping'
+}
+
+export abstract class SvcProcessResult extends Command {
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+  protected checkResult(
+    result: any,
+    serviceType: string,
```

Comment:
Here you can use more specific types: `ProcessExecutionResult` and `ServiceTypes`

Reply:
done